Movie crud

chunchunmaru34 opened the pull request:
Movie CRUD with basic design.

tsuinkotei committed:
Implemented movie add and update function using movie-details.component

tsuinkotei committed:
Added delete movie function and fixed links

chunchunmaru34 committed:
Code brushing

MikhailMartsinovich commented on src/app/movie/actors-list/actors-list.component.html:
```diff
@@ -0,0 +1,17 @@
+      <div>
+        <button (click)="deleteActor(i)" class="btn btn-danger btn-delete-actor" type="button">&#10005;</button>
+      </div>
+      <input class="form-control" [(ngModel)]="actors[i]">
```
зачем теюе здесь [(ngModel)]="actors[i], если у тебя в области видимости есть текущий actor?

MikhailMartsinovich commented on src/app/movie/actors-list/actors-list.component.scss:
```diff
@@ -0,0 +1,9 @@
+.actors-list {
+  button {
```
старайся не использовать одиночные селекторы html тэгов, типа button, либо button.class либо просто .class? т к селекторы тэгов может вызывать много проблем с оверрайдом стилей

MikhailMartsinovich commented on src/app/movie/movie-details/movie-details.component.html:
```diff
@@ -1,16 +1,46 @@
-      <img [src]="movie.posterUrl"/>
+    <div class="col-sm-5 movie-poster form-group">
+      <img *ngIf="movie.posterUrl" class="img-fluid" [src]="movie.posterUrl"/>
+      <img *ngIf="!movie.posterUrl" class="img-fluid" src="http://via.placeholder.com/400x500"/>
```
вынеси src в переменную/константу

AliakseiLabkovich committed:
-Poster placeholder url was places to variable
-Improved actors list html structure
-Added custom checking to ngFor directive, to avoid losing focus after typing 1 symbol

[(ngModel)]="actors[i]" will remains unchanged because actors is array of strings

chunchunmaru34 committed:
Merge branch 'modules_setup' into movie-crud
